docs/admin/deployment.rst

```diff
@@ -99,6 +99,21 @@ you create:
    production.
    Adds some additional django apps that can be helpful during day to day development.
 
+``EMAIL_HOST_PASSWORD``
+   Optional. Password for the SMTP connection (default: ``'apikey'``, no e-mail will be sent if it is set to an empty string).
```

Comment:
Nit: why do we need the "no e-mail will be sent if..." piece here, but not in `EMAIL_HOST` for example?

Should we instead remove the `Optional.` bit (and same for `EMAIL_HOST_USER`)?